src/CoreWCF.Primitives/src/CoreWCF/Dispatcher/SecurityServiceDispatcher.cs

```diff
@@ -0,0 +1,593 @@
+﻿using CoreWCF.Channels;
+using CoreWCF.Configuration;
+using CoreWCF.Description;
+using CoreWCF.Runtime;
+using CoreWCF.Security;
+using CoreWCF.Security.Tokens;
+using System;
+using System.Collections.Generic;
+using System.Security;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Schema;
+
+namespace CoreWCF.Dispatcher
+{
+    public class SecurityServiceDispatcher : IServiceDispatcher
+    {
+        private IRequestReplyCorrelator _requestReplyCorrelator;
+        private BindingContext bindingContext;
```

Comment:
Looking through this list of fields, I'm seeing quite a mismatch between WCF. The following fields are absent. Can you explain why they aren't needed here.
        ChannelBuilder channelBuilder;
        bool extendedProtectionPolicyHasSupport;
        ISecurityCapabilities securityCapabilities;
        EndpointIdentity identity;

Reply:
I see channelBuilder and identity have been put back, but what happened to extendedProtectionPolicyHasSupport and securityCapabilities?